src/components/structures/RoomView.tsx

```diff
@@ -972,7 +987,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
         if (this.unmounted) return;
         // Attach a widget store listener only when we get a room
         WidgetLayoutStore.instance.on(WidgetLayoutStore.emissionForRoom(room), this.onWidgetLayoutChange);
-        this.onWidgetLayoutChange(); // provoke an update
+        // this.onWidgetLayoutChange(); // provoke an update // Done with checkwidget now TODO-maximise_widget remove comment after review
```

Comment:
Ideally the code in the PR is a version that could be merged, so typically better to keep temporary commented lines in a local stash or something like this.